yb-voyager/src/srcdb/oracle.go

```diff
@@ -142,6 +142,10 @@ func (ora *Oracle) GetAllTableNamesRaw(schemaName string) ([]string, error) {
 		if err != nil {
 			return nil, fmt.Errorf("error in scanning query rows for table names: %v", err)
 		}
+		if strings.HasPrefix(tableName, "VOYAGER_LOG_MINING_FLUSH_") {
```

Comment:
I was just thinking should we use the exact table name here or not.

Let's say two separate migration(different uuid) are running for the same schema, in that case we don't want any migration's the flush table to comeup, right?

So then it is fine.

Reply:
yeah right, now with this we will ignore all such tables prefixed with `VOYAGER_LOG_MINING_FLUSH`